core/quantity/src/Interfaces.ts

```diff
@@ -106,6 +105,17 @@ export interface UnitsProvider {
   getConversion(fromUnit: UnitProps, toUnit: UnitProps): Promise<UnitConversionProps>;
 }
 
+/** This interface is implemented by the class that is responsible for locating units by name or label and providing conversion values between units.
+ * The methods to be implemented are async allowing the UnitsProvider to query the backend when necessary to look up unit definition and conversion rules.
+ * @beta
+ */
+export interface UnitsProviderSync {
+  // findUnit(unitLabel: string, schemaName?: string, phenomenon?: string, unitSystem?: string): Promise<UnitProps>;
+  // getUnitsByFamily(phenomenon: string): Promise<UnitProps[]>;
```

Comment:
I'm fine with leaving these commented out, the use case for this is very specific, and we'll only open it up if users really need it (not want)

Reply:
I removed since they were a bit confusing having promises in the definitions within the "sync" version.